docs/general/ddl/data-types.rst

```diff
@@ -1281,13 +1290,7 @@ timestamp into UTC prior to insertion. Contrast this with the behavior of
     In the absence of an explicit :ref:`WITH TIME ZONE
     <type-timestamp-with-tz>` or :ref:`WITHOUT TIME ZONE
     <type-timestamp-without-tz>`, CrateDB will interpret ``TIMEZONE`` as an
```

Comment:
Should this phrase ``TIMESTAMP`` instead of ``TIMEZONE``?

Reply:
yes, you are right.